app/javascript/controllers/locked_poster_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
Also there is already a `locked-media-poster` controller. Can we consolidate these?

Reply:
No, they are different because media may have a poster to show, but for a pdf, we always show the lock icon.

Reply:
Wait shouldn't we always show the lock icon, even if there is a poster?

Reply:
I don't know what the behavior is expected for media. My inclination is not to change anything not related to the ticket.

Reply:
Presently media displays the poster, not a lock.
![Uploading Screenshot 2025-02-18 at 4.05.34 PM.png…]()

Reply:
I am going to put this in post standup for tomorrow but I don't think it's a blocker. We can always consolidate in another ticket.